test/parallel/test-stream2-readable-wrap.js

```diff
@@ -81,10 +67,5 @@ runTest(100, false, function() { return Buffer.allocUnsafe(100); });
 runTest(10, false, function() { return Buffer.from('xxxxxxxxxx'); });
 runTest(1, true, function() { return { foo: 'bar' }; });
 
-var objectChunks = [ 5, 'a', false, 0, '', 'xyz', { x: 4 }, 7, [], 555 ];
-runTest(1, true, function() { return objectChunks.shift(); });
-
-process.on('exit', function() {
-  assert.equal(testRuns, completedRuns);
-  console.log('ok');
-});
+const objectChunks = [ 5, 'a', false, 0, '', 'xyz', { x: 4 }, 7, [], 555 ];
+runTest(1, true, function() { return objectChunks.shift(); });
```

Comment:
No newline at the end of the file.

![image](https://cloud.githubusercontent.com/assets/15943089/23832711/4324d3b6-0732-11e7-95d1-410bb03b173a.png)